packages/renderer/src/lib/task-manager/TaskManager.svelte

```diff
@@ -44,7 +44,7 @@ const taskWordPlural = $derived(selectedItemsNumber > 1 ? 'tasks' : 'task');
 {#if showTaskManager}
   <div
     bind:this={outsideWindow}
-    class="fixed bottom-8 right-4 bg-[var(--pd-modal-bg)] min-h-96 h-3/4 w-[calc(100%-52px-theme(width.leftnavbar))] z-40 border border-[var(--pd-modal-border)] rounded-md  shadow-xl shadow-black">
+    class="fixed bottom-8 right-4 bg-[var(--pd-modal-bg)] min-h-96 h-3/4 w-[calc(100%-52px-(var(--width-leftnavbar)))] z-40 border border-[var(--pd-modal-border)] rounded-md  shadow-xl shadow-black">
```

Comment:
issue: Not sure what change is responsible, but the resize is not something expected?

![image](https://github.com/user-attachments/assets/2133a40d-2cbd-4fb5-a046-7533573105cf)

Reply:
should be fixed

Reply:
I had to move the width definition from the legacy/(now old) config file to the css file theme() definition so after that a css variable is being defined